twitter/Models/Tweet.h

```diff
@@ -0,0 +1,33 @@
+//
+//  Tweet.h
+//  twitter
+//
+//  Created by Joey R. Hernandez Perez on 6/29/20.
+//  Copyright © 2020 Emerson Malca. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "User.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Tweet : NSObject
+
+@property (nonatomic, strong) NSString *idStr; // For favoriting, retweeting & replying
+@property (nonatomic, strong) NSString *text; // Text content of tweet
+@property (nonatomic) int favoriteCount; // Update favorite count label
+@property (nonatomic) BOOL favorited; // Configure favorite button
+@property (nonatomic) int retweetCount; // Update favorite count label
+@property (nonatomic) BOOL retweeted; // Configure retweet button
+@property (nonatomic, strong) User *user; // Contains Tweet author's name, screenname, etc.
+@property (nonatomic, strong) NSString *createdAtString; // Display date
```

Comment:
ooooh. great learning opportunity here. Think about what you would want this to be as a user of your own api.